tests/integration/loopback_test.yaml

```diff
@@ -0,0 +1,34 @@
+description: Delete the loopback interface
+interfaces:
+- name: lo
+  type: loopback
+  state: absent
+description: Set the address to 192.0.2.251/24 and 2001:db8:1::1/64 for loopback device
```

Comment:
`Add the address 192.0.2.251/24 and 2001:db8:1::1/64 to loopback device.`

Because we always preserve 127.0.0.1 and ::1 for loopback interface. So set is wrong here.

Reply:
I updated it, thanks.